packages/filter/README.md

```diff
@@ -0,0 +1,135 @@
+# @loopback/filter
```

Comment:
Which name is better? 

- `@loopback/filter`
- `@loopback/query-filter`
- `@loopback/repository-filter`

Reply:
I prefer `@loopback/filter` as the filter is use both when working with queries and repositories. Prefixing it with `query-` or `repository-` may create confusion that the filter only applies when working with either one of those concepts.

Reply:
Good discussion!

I vote for `@loopback/filter`.